### ISSUE TITLE
Adding things such as graph and wind maps to the ui

### ISSUE THREAD

**Issue by Slambot01** [Mohit5Upadhyay/WeatherApplication#24]
i want to add things such as graph and maps which will enhance the user experience.

**Comment by Pujitha254** [comment]
@Mohit5Upadhyay i wanted to add things such as graph and wind maps to the ui so that it will be more user friendly and interactive

**Comment by Mohit5Upadhyay** [comment]
Yes , @Pujitha254 and @Slambot01  go ahead to implement graphs and maps to UI but they are dynamically changes as per the weather condition.

**Comment by Slambot01** [comment]
can u please assign it under gssoc ext and hacktoberfest label?

**Comment by Pujitha254** [comment]
Thankyou for assigning to me . can u please assign it under gssoc-ext & hacktoberfest label?

**Comment by Slambot01** [comment]
can u please assign me the PR ans also add the labels and levels?
